src/VirtualCollection.vue

```diff
@@ -174,6 +180,13 @@ export default {
             }
         }
     },
+    mounted() {
+        this.resizeObserver = new ResizeObserver(this.onContainerResized)
```

Comment:
`ResizeObserver` is an experimental feature in browser, which is not fully supported yet.
It would be better to make a fallback for it.